### DISCUSSION TITLE
Setting nicknames

### DISCUSSION THREAD

**Discussion by kowalskies** [kibook/poodlechat#11] [Q&A] [answered]
Hi we just added this to a new server and its running in default config but the /nick command doesnt work for anyone but admins. Can you tell me how to fix it if possible? None of the files are changed yet.

**Comment by kibook** [comment, marked as answer]
You probably need to add the following to server.cfg:

```
add_ace builtin.everyone command.nick allow
```

That will allow all players to use the `nick` command. The reason only admins can is probably that in most default server.cfg files, you'll have this:

```
add_ace group.admin command allow
```

That allows players in the `group.admin` principal to run any command.

**Reply by kowalskies** [reply]
Thanks for the quick response, I'll forward this to the admin!
